spec/features/agents/agent_can_create_rdv_with_wizard_spec.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-describe "Agent can create a Rdv with wizard", js: true do
+describe "Agent can create a Rdv with wizard" do
```

Comment:
Je passe le `js: true` au niveau des `it` pour plus de clarté, et aussi car la spec que j'ajoute n'a pas besoin du JS !